config/config.go

```diff
@@ -37,23 +37,24 @@ var (
 
 	// CFG Houses configuration meta
 	CFG = cfgs{
-		CloudAPIProtocol:  newCfg("cloud.api.protocol", "https"),
-		CloudAPIPort:      newCfg("cloud.api.port", "443"),
-		CloudWSProtocol:   newCfg("cloud.api.ws_protocol", "wss"),
-		CloudAPIToken:     newCfg("cloud.api.token", ""),
-		Context:           newCfg("context", ""),
-		Contexts:          newCfg("contexts", ""),
-		LocalHouston:      newCfg("local.houston", ""),
-		LocalOrbit:        newCfg("local.orbit", ""),
-		PostgresUser:      newCfg("postgres.user", "postgres"),
-		PostgresPassword:  newCfg("postgres.password", "postgres"),
-		PostgresHost:      newCfg("postgres.host", "postgres"),
-		PostgresPort:      newCfg("postgres.port", "5432"),
-		ProjectDeployment: newCfg("project.deployment", ""),
-		ProjectName:       newCfg("project.name", ""),
-		ProjectWorkspace:  newCfg("project.workspace", ""),
-		WebserverPort:     newCfg("webserver.port", "8080"),
-		ShowWarnings:      newCfg("show_warnings", "true"),
+		CloudAPIProtocol:   newCfg("cloud.api.protocol", "https"),
+		CloudAPIPort:       newCfg("cloud.api.port", "443"),
+		CloudWSProtocol:    newCfg("cloud.api.ws_protocol", "wss"),
+		CloudAPIToken:      newCfg("cloud.api.token", ""),
+		Context:            newCfg("context", ""),
+		Contexts:           newCfg("contexts", ""),
+		LocalHouston:       newCfg("local.houston", ""),
+		LocalOrbit:         newCfg("local.orbit", ""),
```

Comment:
I know this is only a code formatting change, but should we add a todo to remove `LocalOrbit` throughout the CLI?  Is this still in use?  It could potentially make the codebase easier to work with depending on how much of this unused code is still in here.

Reply:
@adam2k i can remove it in separate PR ok?